internal/log/level.go

```diff
@@ -0,0 +1,46 @@
+package log
+
+import "strings"
+
+type Level uint8
+
+const (
+	DEBUG Level = iota
+	INFO
+	DEBUGL
+	WARN
+	ERROR
+	FATAL
+)
+
+func (lv Level) String() string {
+	switch lv {
+	case DEBUG:
+		return "debug"
```

Comment:
Please use UpperCase for print Log Levels

Reply:
thankyou for your review